crates/rerun/src/run.rs

```diff
@@ -249,32 +250,37 @@ async fn run_impl(
 
     // Where do we get the data from?
     let rx = if let Some(url_or_path) = &args.url_or_path {
-        let path = std::path::Path::new(url_or_path).to_path_buf();
-        if path.exists() || url_or_path.ends_with(".rrd") {
-            re_log::info!("Loading {path:?}…");
-            load_file_to_channel(&path).with_context(|| format!("{path:?}"))?
+        if url_or_path.starts_with("http") {
+            re_viewer::stream_rrd_from_http::stream_rrd_from_http_to_channel(url_or_path.clone())
         } else {
-            // We are connecting to a server at a websocket address:
-            let mut rerun_server_ws_url = url_or_path.clone();
-            if !rerun_server_ws_url.contains("://") {
-                rerun_server_ws_url = format!("{}://{rerun_server_ws_url}", re_ws_comms::PROTOCOL);
-            }
-            if args.web_viewer {
-                return host_web_viewer(rerun_server_ws_url).await;
+            let path = std::path::Path::new(url_or_path).to_path_buf();
+            if path.exists() || url_or_path.ends_with(".rrd") {
```

Comment:
It's kind of strange that any string that doesn't exist as a path be treated as a websocket...
```
$ cargo run -p rerun foo.png
[2023-03-17T11:48:42Z INFO  re_viewer::remote_viewer_app] Connecting to WS server at "foo.png"…
[2023-03-17T11:48:42Z INFO  re_ws_comms::client] Connecting to "foo.png"…
[2023-03-17T11:48:42Z ERROR re_ws_comms::client] Connection error: URL error: URL scheme not supported
```